packages/p2p/stores/advertiser-page-store.js

```diff
@@ -0,0 +1,205 @@
+import { action, observable } from 'mobx';
+import { buy_sell } from '../src/constants/buy-sell';
+import { getShortNickname } from 'Utils/string';
+import { height_constants } from 'Utils/height_constants';
+import { localize } from 'Components/i18next';
+import { requestWS } from 'Utils/websocket';
+
+export default class AdvertiserPageStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+        this.general_store = this.root_store.general_store;
+    }
+
+    @observable active_index = 0;
+    @observable advertiser_info = {};
+    @observable ad = null;
+    @observable adverts = [];
+    @observable counterparty_type = buy_sell.BUY;
+    @observable error_message = '';
+    @observable form_error_message = '';
+    @observable is_loading = true;
+    @observable is_submit_disabled = true;
+    @observable show_ad_popup = false;
+    @observable submitForm = () => {};
+
+    height_values = [
+        height_constants.screen,
+        height_constants.advertiser_page_content,
+        height_constants.core_header,
+        height_constants.core_footer,
+        height_constants.filters,
+        height_constants.filters_margin,
+        height_constants.page_overlay_header,
+        height_constants.page_overlay_content_padding,
+        height_constants.table_header,
+        height_constants.tabs,
+    ];
+    item_height = 56;
+    props = {};
```

Comment:
Should we make this `observable` since some `computed` values are referencing this?